10.11/Dockerfile.fedora

```diff
@@ -37,10 +37,13 @@ LABEL summary="${SUMMARY}" \
 
 EXPOSE 3306
 
-# This image must forever use UID 27 for mysql user so our volumes are
-# safe in the future. This should *never* change, the last test is there
-# to make sure of that.
-RUN INSTALL_PKGS="policycoreutils rsync tar gettext hostname bind-utils groff-base" && \
+# This image must forever use UID 27 for mysql user so our volumes are safe in the future. This should *never* change.
+# Instead of relying on the DB server package, we will do the setup ourselves before any package is installed
+RUN /usr/sbin/groupadd -g 27 -o -r mysql && \
+    /usr/sbin/useradd -M -N -g mysql -o -r -d ${HOME} -s /sbin/nologin -c "MySQL Server" -u 27 mysql && \
+    test "$(id mysql)" = "uid=27(mysql) gid=27(mysql) groups=27(mysql)" && \
+    \
```

Comment:
Typo here.

Reply:
Where exactly?

In the end I decided to NOT add extra `RUN` phase just for user addition, so I removed the `RUN` from line just after this one and added a blank line with just `    \` to at least visually split the blocks for human readers.

Reply:
Only visual split is not needed. The content is still readable even there is no split.